packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -293,6 +293,28 @@ class TransactionService {
             if (fid > 0) {
               tr.experience.fid = fid
             }
+
+            let longtaskCount = 0
+            let longtaskDuration = 0
+            let longtaskMax = 0
+            for (let i = 0; i < tr.spans.length; i++) {
+              const span = tr.spans[i]
+              if (span.type === LONG_TASK) {
+                longtaskCount++
+                const dur = span.duration()
+                longtaskDuration += dur
+                if (dur > longtaskMax) {
+                  longtaskMax = dur
+                }
+              }
+            }
+            if (longtaskCount > 0) {
+              tr.experience.longtask = {
```

Comment:
These have to be shortened fields. For the new proposals, we tend to follow the shortened formats. So we need something like

```js
tr.experience.lt = {
     d: longtaskDuration, // duration is already in mapping
     ct: longtaskCount,
     lo: longestLongTask
}
```

Reply:
I think we should use the shorthand if the fields have well-defined abbreviations like `cls` for others it seems to me it's a diminishing return.

Reply:
Not really,  its not just for optimisation. We have to keep it consistent across the spec. Having all other fields as short-hand and having these ones with full form doesn't make any sense. 

I am leaning towards having them as short hand fields, If you are still not convinced, let's hear it from the server team.